github/issues.go

```diff
@@ -131,6 +132,9 @@ func (s *IssuesService) listIssues(u string, opt *IssueListOptions) ([]Issue, *R
 		return nil, nil, err
 	}
 
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeReactionsPreview)
```

Comment:
Very interesting!  Yes, it sounds like a bug on GitHub's side.  Would you mind reporting it, Dmitry?  I'm heading off to OSCON.  Thanks!